api-compact.yaml

```diff
@@ -85,9 +91,11 @@ x-amazon-apigateway-integration: &apigateway_integration
   type: "http"
   requestParameters:
     integration.request.header.Authorization: "method.request.header.Authorization"
+    integration.request.header.Host: "${stageVariables.enterprise_host}"
```

Comment:
Seems like we should be using the `X-Forwarded-Host` header, set to the gateway's custom domain name, which might be more appropriate here and is something we can change trivially later - might be worth creating a new globally-available stage variable called `gateway_host` or something that we can configure as needed? (I imagine that could be a separate PR against api-manager that I can take on if it's valuable.)

Another question is around pathing. The path of the enterprise endpoint through the gateway is something like `api.gateway.host/enterprise/v1/endpoint`, while through the origin is `edxapp.host/enterprise/api/v1/endpoint`. (Note the `api` in the path in the second example.) So this isn't just host rewriting, it's also path rewriting right?